src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -179,6 +182,7 @@ public boolean shouldTraverse(NodeTraversal nodeTraversal, Node n, Node parent)
     private int scriptNodeCount = 0;
     private List<Node> moduleExportRefs = new ArrayList<>();
     private List<Node> exportRefs = new ArrayList<>();
+    Map<String, Integer> propertyExportRefCount = new HashMap<>();
```

Comment:
You can often replace a `Map<String, Integer>` with a `Multiset<Key>` and I think this is a case where that would work well, but I'm not sure. Take a look and if it switching to a Multiset is a lot of work then nevermind but if it turns out to be easy let's do it.

http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/collect/Multiset.html